scripts/main/mapview.js

```diff
@@ -296,6 +328,7 @@ mapview.open = function (albumID = null) {
 		 */
 		const successHandler = function (data) {
 			addPhotosToMap(data);
+			setTrack(data);
```

Comment:
I wonder if it is the right approach to have two methods here which both take the retrieved album as an argument. In particular, both methods do not only add content to the map, but zoom and pan the map such that the map covers the bounding box of the content. At least the last step is redundant, because it is executed by both methods.

IMHO, either we should combine everything into one method `addContentToMap` or we should have three methods and pull out the zooming step in an extra method, i.e. `addPhotosToMap`, `addTrackToMap`, `zoomToBoundingBox`.

Reply:
I merged both methods into `addContentToMap`, an `updateZoom` was already there and I just updated it.